3rdparty/haskell/BUILD.ghc-lib-parser

```diff
@@ -0,0 +1,95 @@
+load("@io_tweag_rules_haskell//haskell:haskell.bzl", "haskell_library")
```

Comment:
I assume we're going to fix this upstream?

Reply:
Hopefully but it’s somewhat tricky to do, I already opened an upstream issue about this a while back https://github.com/tweag/rules_haskell/issues/720